WexHomework4/WexHomework4/Program.cs

```diff
@@ -0,0 +1,109 @@
+﻿
+
+// заполняем массив рандомными числами от -100 до 100
+static void IntializeArray(int[] array, int cofficient)
+{
+    Random rnd = new Random();
+   
+
+    for (int step = 0; step < array.Length; step++)
+    {
+        array[step] = rnd.Next(-100, 100);
+        //Console.WriteLine(array[step]);
+    }
+    
+}
+//находим экстримумы мин или макс
+static int extremumCalculation(int[] arrayExtr,string extremum)
+{    
+    int extr = arrayExtr[0];
+    for (int e = 0; e < arrayExtr.Length; e++)
+    {
+        if (extremum == "min")
+        {
+            if (arrayExtr[e]<extr) extr= arrayExtr[e];
+        }
+        else
+        {
+            if (arrayExtr[e] > extr) extr = arrayExtr[e];
+        }
+    }
+ 
+    return extr;
+}
+//возращает массив после перестановки мин и макс местами
+static int[] finalArray(int[] array3,int valueMin,int valueMax)
+{
+    
+    for (int i = 0; i < array3.Length; i++)
+    {
+
+        if (array3[i] == valueMin)
+        {
+            array3[i] = valueMax;
+
+        }
+        else if (array3[i] == valueMax)
+        {
+            array3[i] = valueMin;
+        }
+    }
+    return array3;
+}
+
+//находит индексы где расположены все мин или все макс значения
+
+static Array findExtremumPositions(int[] arrays, string extremum2)
```

Comment:
`you can use Array.IndexOf()`

Reply:
My method returns all indexes if the array has few values which are equal to min(or max) value but according to https://docs.microsoft.com/en-us/dotnet/api/system.array.indexof?view=net-6.0 Array.IndexOf() can return its first occurrence only. So it is not usable for my method.